examples/simple/main.cc

```diff
@@ -11,18 +11,19 @@ namespace
 {
 void initTracer()
 {
-  auto exporter  = std::unique_ptr<sdktrace::SpanExporter>(new StdoutExporter);
+  auto exporter = std::unique_ptr<sdktrace::SpanExporter>(new StdoutExporter);
+  // Specify the Span Processor: SimpleSpanProcessor forwards spans directly to the exporter
```

Comment:
Span Processor -> SpanProcessor (remove the space)

I also wonder if this is self-explanatory?

Reply:
agreed, removed the comment